Makefile

```diff
@@ -19,6 +19,9 @@ RUN = argon2
 BENCH = bench
 GENKAT = genkat
 
+# Increment on an ABI breaking change
+ABIVERSION = 1
```

Comment:
This is bumping the ABI version to 1 for no clear reason. Please keep in mind that every time this is bumped, all dependent packages need to be relinked.